### ISSUE TITLE
Luminosity ratio for "Hide" button is 2.5:1 which is less than 3:1

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1132]
**Repro Steps**
1. Launch 'Calculator'.
2. Home screen will be displayed.​
3. Navigate through Menu using keyboard and select 'Graphing'. 
4. Navigate to hide button.
5. Check luminosity ratio for hide button using color contrast analyzer tool. ​
6. Observe the issue. 

**Actual Result:**
Luminosity ratio for "Hide" button is 2.5:1 which is less than 3:1.

**Expected Result:**
Luminosity ratio for "Hide" button should be minimum 3:1.

![image](https://user-images.githubusercontent.com/1414423/78102533-63d32e00-739f-11ea-95ab-3e278d62a1e6.png)

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
